contracts/proxy/upgradeable/UpgradeableOwnableProxy.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { UpgradeableOwnableProxyBase } from './UpgradeableOwnableProxyBase.sol';
+import { Proxy } from '../Proxy.sol';
+import { OwnableInternal } from '../../access/OwnableInternal.sol';
+import { OwnableStorage } from '../../access/OwnableStorage.sol';
+import { UpgradeableOwnableProxyStorage } from './UpgradeableOwnableProxyStorage.sol';
+
+contract UpgradeableOwnableProxy is UpgradeableOwnableProxyBase {
```

Comment:
Rename to `UpgradeableProxyOwnable` for parallel structure with `ManagedProxyOwnable`, etc.